### ISSUE TITLE
vipsheader hangs on AFL generated input

### ISSUE THREAD

**Issue by Kyle-Kyle** [libvips/libvips#1377]
base64 encoded image content: UEYgNiAgMCAjRgEAGwAAAAMACzxa//8AABQAgP//ABEIAPsBIBvtIooA+vA=

**Comment by Kyle-Kyle** [comment]
command line:
`./.libs/vipsheader <input_image>`

**Comment by jcupitt** [comment]
Hello @Kyle-Kyle ,

I see this too with:

```
$ echo UEYgNiAgMCAjRgEAGwAAAAMACzxa//8AABQAgP//ABEIAPsBIBvtIooA+vA= | base64 -d > x
$ vipsheader x
```

It looks like it's trying to read the ppm header and failing. Thanks for reporting this!

**Comment by jcupitt** [comment]
OK, fixed in 8.8, this will be in 8.8.2. Thank you very much for reporting this dumbness!

**Comment by lovell** [comment]
libvips compiled using the `--without-ppm` flag is not affected by this.
